packages/client/types/constEnums.ts

```diff
@@ -170,8 +170,16 @@ export const enum MeetingLabels {
 export const enum PokerCards {
   HEIGHT = 175,
   WIDTH = 125,
-  OVERLAP = 96
-
+  OVERLAP = 96,
+  MAX_VALUE = 2147483647,
+  // the angle in degrees from the middle of the deck to a deck edge (0-90)
+  TILT = 8,
```

Comment:
@mattkrick I set the radius and tilt so that the Fibonacci deck looked good on mobile (~360px). I considered making these responsive so that the cards could breathe a little better at desktop breakpoints. Worth pursuing?

Reply:
absolutely.
could you just fill in the logic to this function?
```ts
calculateTiltAndRadius = (totalCards: number, maxWidth: number) => {
  // do stuff
  return {tilt, radius}
}
```

Reply:
Oh, something like this, where we could play with 2 const enums based on a breakpoint?
```js
const calculateTiltAndRadius = (totalCards: number, maxWidth: number) => {
    // RADIUS = 900
    // RADIUS_MOBILE = 1200
    // TILT = 12
    // TILT_MOBILE = 8
    const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
    const tilt = isDesktop ? PokerCards.TILT : PokerCards.TILT_MOBILE
    const radius = isDesktop ? PokerCards.RADIUS : PokerCards.RADIUS_MOBILE
    // calculate based on const enums
    return {tilt, radius}
  }
```
The fanning effect is nice when there’s room. 😄

Reply:
ah gotcha, so the number of cards doesn't affect anything?

Reply:
ah, good point, we could play with the width if the number of cards is high or low, which could save space at the mobile breakpoint, I’ll simmer on that, but I did look at that cheap trick of just setting 2 settings based on the breakpoint

Reply:
something that might be fun is to make a table, maybe in excel?
you could do something like:

| width| totalCards | tilt | radius |
| 300 | 10 | 8  | 1000 |
| 400 | 10 | 10 | 1200

and after a few samplings (maybe 20?), we could derive a function from that

Reply:
I made a side branch and backed up to the commit before I yanked the hotkeys so that I can play with it easier. I’ll see if I can find a pattern.